docs/updating/update-to-40.md

```diff
@@ -405,9 +405,9 @@ While `⠿` is now a default, the icon can still be configured by the integrator
 
 ### A new default lists plugin coming
```

Comment:
Needs to be moved to update to 41.

Reply:
Unless this is just link updates?

Reply:
There will we a separate entry in Update to 41.x about the lists.